docs/source/interfaces/indirect/Indirect Data Manipulation.rst

```diff
@@ -0,0 +1,268 @@
+.. _interface-indirect-data-manipulation:
```

Comment:
I think we should put this into a new "inelastic" category. What do you think @SilkeSchomann and @SpencerHowells?

Reply:
@AnthonyLim23 Yes, it would make sense to have this in a new "inelastic" category.

Reply:
I'll make that change

Reply:
@AnthonyLim23 Or both?

Reply:
I think we should put this into the new inelastic category. We should also move data analysis into it too (with the new name "curve fitting"), but this can be a separate issue

Reply:
I have been workin on some later issue stages off of this branch, Moving these interfaces and renaming them apropriatly would make rebaseing them more difficalt. I think it would be best to finish those changes then move the interface and rename them.

Reply:
ok, can we have an issue for it?

Reply:
Issue made #34676